Singularity.3.6.0

```diff
@@ -0,0 +1,58 @@
+BootStrap: docker
+From: ubuntu:16.04
+
+%labels
+  Maintainer Jeremy Nicklas
+  R_Version 3.5.3
```

Comment:
Need to bump the version here, as well as in the main `Singularity` file.

Reply:
Done! Thanks for picking that up.